inc/saiswitch.h

```diff
@@ -2776,6 +2789,17 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_ECMP_MEMBER_COUNT,
 
+    /**
+     * @brief Switch MDIO access type
+     *
+     * set mdio access type (clause 22 or clause 45) before calling sai_switch_mdio_read_fn and sai_switch_mdio_write_fn
```

Comment:
why can't the platform decide to choose between cl22 or cl45 for the mdio r/w? why should a higher SAI layer tell which one to choose? i am not able to understand the usecase. Can you add more to the PR description?